src/ImageSequencer.js

```diff
@@ -102,7 +102,7 @@ ImageSequencer = function ImageSequencer(options) {
     return this;
   }
 
-  function insertSteps(image, index, name, o) {
+  function insertSteps() {
```

Comment:
Hmm, was this at some point to allow inserting a step at a different index? And same for `removeSteps()` above? @Divy123 

Thank you!

Reply:
I think we are managing that through arguments array.